lib/node_modules/@stdlib/math/base/special/sqrt1pm1/benchmark/cpp/native/benchmark.c

```diff
@@ -0,0 +1,136 @@
+/**
```

Comment:
@Pranavchiku I believe this file should be in a `/benchmark/c` folder, rather than `benchmark/cpp`.